code/datums/ruins/lavaland.dm

```diff
@@ -168,13 +168,15 @@
 	id = "alien-nest"
 	description = "Not even Necropolis is safe from alien infestation. The competition for hosts has locked the legion and aliens in an endless conflict that can only be resolved by a PKA."
 	suffix = "lavaland_surface_alien_nest.dmm"
-	cost = 20
+	cost = 10
+	allow_duplicates = FALSE
 
 /datum/map_template/ruin/lavaland/fountain
 	name = "Fountain Hall"
 	id = "fountain"
 	description = "The fountain has a warning on the side. DANGER: May have undeclared side effects that only become obvious when implemented."
 	suffix = "lavaland_surface_fountain_hall.dmm"
+	allow_duplicates = FALSE
 	cost = 5
```

Comment:
differing order of vars here which i don't actually think this matters but code consistency